pulp_smash/tests/rpm/api_v2/test_copy_mtime.py

```diff
@@ -0,0 +1,93 @@
+# coding=utf-8
+"""Test whether copy of directory will copy the ``mtime``."""
+import os
+import re
+import time
+import unittest
+
+from pulp_smash import api, cli, config, selectors, utils
+from pulp_smash.constants import (
+    REPOSITORY_PATH,
+    RPM_SIGNED_URL,
+    RPM_UNSIGNED_FEED_URL,
+)
+from pulp_smash.tests.rpm.api_v2.utils import gen_distributor, gen_repo
+from pulp_smash.tests.rpm.utils import set_up_module as setUpModule  # noqa pylint:disable=unused-import
+
+_PATH = '/var/lib/pulp/published/yum/https/repos/'
+
+
+class CopyDirectoryStepTestCase(unittest.TestCase):
+    """Test whether copy of directory will copy the ``mtime``."""
+
+    def test_all(self):
+        """Test whether copy of directory will copy the ``mtime``.
+
+        This test targets the following issues:
+
+        * `Pulp Smash #720 <https://github.com/PulpQE/pulp-smash/issues/720>`_
+        * `Pulp #2783 <https://pulp.plan.io/issues/2783>`_
+
+        Do the following:
+
+        1. Create and sync a repository.
+        2. Publish the repository with the option ``generate_sqlite``
+           as True.
+        3. Verify the ``mtime`` for the sqlite files.
+        4. Upload a RPM package to the repository.
```

Comment:
Is this a new package or an existing package already synced?

Reply:
It is a different package. I used as feed url a unsigned package, and to upload a signed package.